### ISSUE TITLE
feat(hubspot): force https config

### ISSUE THREAD

**Issue by shadow81627** [DavidWells/analytics#84]
Allow https to be set true in config, my lighthouse audit fails because this the localhost protocol is http rather than https. I would like to set config to force https connections so Lighthouse security audits pass.

https://github.com/DavidWells/analytics/blob/master/packages/analytics-plugin-hubspot/src/browser.js#L33

**Comment by DavidWells** [comment]
Hey @shadow81627 It looks like the protocol is set to https

```js
const https = protocol === 'https:' || protocol === 'chrome-extension:'
```

What is the error you are seeing?

Edit: nvm I see https://github.com/DavidWells/analytics/blob/master/packages/analytics-plugin-hubspot/src/browser.js#L35

**Comment by shadow81627** [comment]
I was running a localhost server http://local host:3000 in my ci/cd so I think the protocol was getting set to http rather than https.

**Comment by DavidWells** [comment]
I've added a custom option to define the URL where the HubSpot script loads from

You can use this to force HTTPS on localhost

```js
hubSpotPlugin({
  portalId: '234576',
  customScriptSrc: `https://js.hs-scripts.com/234576.js`
})
```

This matches `customScriptSrc` setup from [segment](https://github.com/DavidWells/analytics/tree/master/packages/analytics-plugin-segment#configuration-options-for-browser) and GA plugins

**Comment by DavidWells** [comment]
Just released with `@analytics/hubspot@0.3.0` 🎉
